code/index.html

```diff
@@ -16,29 +16,66 @@
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="images/reed.svg"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Which fish <br> is hiding <br> in the reed?</h1>
 
       <select id="questions">
-        <optgroup label="hair">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
+        
+        <option selected="true" disabled="disabled">Does the fish...</option>    
```

Comment:
I think selected is by default true and disabled as well, so you could reduce your code here. By doing something like this  ` <option selected disabled">Does the fish...</option> `